### ISSUE TITLE
Support for String fields compare  kapacitor rules in Chronograf UI

### ISSUE THREAD

**Issue by chetu** [influxdata/chronograf#865]
Haproxy input plugin field "status"  have   UP, DOWN , OPEN.

**In tick its works:**

```
var period = 10s
var every = 10s
var data = stream
    |from()
        .database('influxdb')
        .retentionPolicy('autogen')
        .measurement('haproxy')
        .groupBy('pxname')
    |window()
        .period(period)
        .every(every)
    |last('status')
        .as('value')

var alert = data
    |alert()
        .message('Haproxy monitor : {{.ID}} : {{ index .Tags "server" }} : {{ index .Tags "pxname" }} is {{ .Level }} ')
        .crit(lambda: "value" == 'DOWN' )
        .stateChangesOnly()
        .id('{{ .Name }}')
        .details(''' Email template ''')

``` 
In above  tick script its working, however chronograf UI not working or not have option to it.

![string_support](https://cloud.githubusercontent.com/assets/395601/22828008/e77b667a-efc0-11e6-8fb2-5c25e42acb26.png)

**Comment by goller** [comment]
Hi @chetu, thanks for writing in this bug! I've been able to reproduce in a unit test.  I'm working on a fix right now and it'll ship with today's release.

**Comment by goller** [comment]
Hi @chetu, I've fixed this in PR #867.

Here is the output tickscript now with your parameters:

```javascript
var db = 'influxdb'

var rp = 'autogen'

var measurement = 'haproxy'

var groupBy = ['pxname']

var whereFilter = lambda: TRUE

var period = 10s

var every = 10s

var name = 'haproxy'

var idVar = name + ':{{.Group}}'

var message = 'Haproxy monitor : {{.ID}} : {{ index .Tags "server" }} : {{ index .Tags "pxname" }} is {{ .Level }} '

var idTag = 'alertID'

var levelTag = 'level'

var messageField = 'message'

var durationField = 'duration'

var outputDB = 'chronograf'

var outputRP = 'autogen'

var outputMeasurement = 'alerts'

var triggerType = 'threshold'

var details = 'Email template'

var crit = 'DOWN'

var data = stream
    |from()
        .database(db)
        .retentionPolicy(rp)
        .measurement(measurement)
        .groupBy(groupBy)
        .where(whereFilter)
    |window()
        .period(period)
        .every(every)
        .align()
    |last('status')
        .as('value')

var trigger = data
    |alert()
        .crit(lambda: "value" == crit)
        .stateChangesOnly()
        .message(message)
        .id(idVar)
        .idTag(idTag)
        .levelTag(levelTag)
        .messageField(messageField)
        .durationField(durationField)
        .details(details)
        .email()

trigger
    |influxDBOut()
        .create()
        .database(outputDB)
        .retentionPolicy(outputRP)
        .measurement(outputMeasurement)
        .tag('alertName', name)
        .tag('triggerType', triggerType)

trigger
    |httpOut('output')
```
... which is now in our test cases!

**Comment by goller** [comment]
Ok, it's merged in and will ship with our release today!
